libs/ml/tests/unit/graph/graph_every_op.cpp

```diff
@@ -114,7 +114,83 @@ void ComparePrediction(GraphPtrType g, GraphPtrType g2, std::string node_name)
   EXPECT_TRUE(prediction.AllClose(prediction2, DataType{0}, DataType{0}));
 }
 
-TYPED_TEST(GraphRebuildTest, graph_rebuild_every_op)
+TYPED_TEST(GraphRebuildTest, graph_rebuild_time_distributed_ops)
```

Comment:
why do we separate out these tests? is this to do with time distributed not working with shape deduction currently? will they be merged back together after?

Reply:
well, the reason for separation was - time-distributed ones failed until I fixed them later. Now I just wonder if it would be better to split that "rebuild-all" approach to a smaller groups or even individual tests. But you're right it looks like a separate task, not related to current change, so I'll just merge it back for now.